src/kakarot/instructions/block_information.cairo

```diff
@@ -122,6 +123,12 @@ namespace BlockInformation {
         Stack.push_uint128(evm.message.env.base_fee);
         jmp end;
 
+        blobhash:
+        let stack = cast([fp - 6], model.Stack*);
+        Stack.pop();
+        Stack.push_uint128(0);
```

Comment:
shouldn't we have a similar strategy as for the blockhashes?

Reply:
see PR description